CHANGELOG.md

```diff
@@ -24,6 +24,10 @@ Here is a template for new release sections
 ### Added
 - Benchmark test (`tests/benchmark_test_inputs/objective_value_exception_equal_annuity`) for in `F0_output.parse_simulation_log` and data stored to `SIMULATION_RESULTS` as well as `OBJECTIVE_VALUE` (#901)
 - Constants `BENCHMARK_TEST_INPUT_FOLDER` and `BENCHMARK_TEST_OUTPUT_FOLDER` in `tests/_constants.py` (#901)
+- Add the argument `return_les` to the function `D0.run_oemof` to return the energy system if set to `True` (#923)
+- Save the content of the lp file into a string in the `dict_values` under `SIMULATION_SETTINGS->OUTPUT_LP_FILE` in server mode (#923)
```

Comment:
I can not recomment saving the LP file in `dict_values`, as it can be really big! Are you sure you want to do this? Try it out with one of the benchmark tests that have constrains or timeseries instead of scalar prices, and run it for a year.

Reply:
I checked - the json file is anyway already 25 MB large, but the LP file is 122 MB large!

Reply:
The idea is to set the LP file in the dict values only for debugging purposes, most of the time it would not be included, except if a user tick a box for example (an advanced user who explicitly wants to have a look at the LP file)

Reply:
I should actually delete the one on the server once I saved the content of the file into an IOstream you are right. On the other hand, I always use the same lp file name, so it will get overwritten.